### ISSUE TITLE
[Tutorial] AltairZ80 Emulator

### ISSUE THREAD

**Issue by pulkitkrishna00** [InstallerLegacy/setup.virtualhub.eu.org#9]
- Homepage: <https://schorn.ch/altair.html>

Information:
- Executable
- Manual installation

Status:

- [x] Tutorial
- [x] Video

Depends on: #4

**Comment by pulkitkrishna00** [comment]
Working on it.

**Comment by pulkitkrishna00** [comment]
Added the tutorial in 84c2dd2.

**Comment by pulkitkrishna00** [comment]
Done in 48e5ad9.

**Comment by pulkitkrishna00** [comment]
Need to update the tutorial with correct information.

**Comment by pulkitkrishna00** [comment]
Added video: https://youtu.be/U0qO_eUB2wQ

**Comment by pulkitkrishna00** [comment]
Need to redo the video with new chanes.

**Comment by pulkitkrishna00** [comment]
Closing this, as it has been decided that a separate issue will be created for video.
